docker/build.sh

```diff
@@ -156,16 +151,38 @@ if [ "$CREATE_RUNNER" = "YES" ]; then
     fi
 fi
 
+rm -rf ./new_wheelhouse
 if [ "$CREATE_WHL" = "YES" ]; then
     if [ "$BUILD_INHOST" = "YES" ]; then
-        mkdir -p ./wheelhouse
-        cp $(pwd)/${DALI_BUILD_DIR}/nvidia* ./wheelhouse/
+        mkdir -p ./new_wheelhouse
+        cp $(pwd)/${DALI_BUILD_DIR}/nvidia* ./new_wheelhouse/
     else
         export CONTAINER="extract-tmp"
         docker create --name "${CONTAINER}" ${BUILDER_WHL}
-        rm -rf ./wheelhouse
-        docker cp "${CONTAINER}:/wheelhouse/" "./"
+        docker cp "${CONTAINER}:/wheelhouse/" "./new_wheelhouse"
         docker rm -f "${CONTAINER}"
     fi
 fi
+
+# Build DALI TF plugin
+export CUSTOM_OP_BUILDER_CLEAN_IMAGE_NAME="tf_custom_op_builder_${PYVER}_clean"
+docker build -t ${CUSTOM_OP_BUILDER_CLEAN_IMAGE_NAME} --build-arg "PYVER=${PYVER}" --build-arg "PYV=${PYV}" -f docker/Dockerfile.customopbuilder.clean docker
+
+export CUSTOM_OP_BUILDER_IMAGE_NAME="tf_custom_op_builder_${PYVER}_tf_build"
+export CUSTOM_OP_BUILDER_CONTAINER="${CUSTOM_OP_BUILDER_IMAGE_NAME}_container"
+
+mkdir -p dali_tf_plugin/whl
+cp ./new_wheelhouse/*.whl dali_tf_plugin/whl/
```

Comment:
Why we need this temporary dir? Can't we use `mktemp -d` and pass this tmp dir as an argument to the docker?

Reply:
done

Reply:
Created tmp dir. The argument to docker is not possible because you need to have all the files within the scope of the docker context (the directory where you invoke the docker command)